ReversePolishNotationCalculator/TB1_Program.cs

```diff
@@ -0,0 +1,39 @@
+Console.WriteLine(Calculate("2 4 5 + *")); // Should be 18
+Console.Write(Calculate("2 3 + 5 *")); // Should be 25
+
+double Calculate(string expression)
```

Comment:
```suggestion
double Evaluate(string expression)
```

Výrazy se spíše nepočítají, ale vyhodnocují.